producer/eapAkaPrimePrf_test.go

```diff
@@ -0,0 +1,44 @@
+package producer
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+type testEAPAKAPrimeCase struct {
+	Identity    string `json:"Identity"`
+	NetworkName string `json:"Networkname"`
+	RAND        string `json:"RAND"`
+	AUTN        string `json:"AUTN"`
+	IK          string `json:"IK"`
+	CK          string `json:"CK"`
+	RES         string `json:"RES"`
+	CKPrime     string `json:"CKPrime"`
+	IKPrime     string `json:"IKPrime"`
+	K_encr      string `json:"K_encr"`
+	K_aut       string `json:"K_aut"`
+	K_re        string `json:"K_re"`
+	MSK         string `json:"MSK"`
+	EMSK        string `json:"EMSK"`
+}
+
+func TestEAPAKAPrimePrf(t *testing.T) {
+	testcases := []string{"testcase1.json", "testcase2.json", "testcase3.json", "testcase4.json"}
```

Comment:
The fields in json are fixed. You can define a struct for that and fill the test data in struct here. So, no need to read the data from file.
For example:
```
	testCases := []struct {
		identity     string
		networkname string
	}{
		{
			"0555444333222111",
			"WLAN",
		},
		{
			"0555444333222111",
			"HRPD",
		},
...
```

Reply:
Done the modification according to the comment.